src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Properties/launchSettings.json

```diff
@@ -28,6 +28,7 @@
       "commandName": "Project",
       "dotnetRunMessages": "true",
       "launchBrowser": true,
+      "hotReloadProfile": "aspnetcore",
```

Comment:
Do we need to track an item to add a template option to configure this?

Reply:
What sort of template option?

Reply:
Nevermind. Just realized we probably don't want users configuring this themselves. We'll figure out automagically based on the template type.